### PR TITLE
re-add ovirt_provision_plugin (RPM)

### PR DESCRIPTION
**Pull request by mmoll**
This should be built into:
* [x] Nightly
* [x] 1.12
* [x] 1.11
* [ ] 1.10

see https://github.com/theforeman/ovirt_provision_plugin/pull/8

### CODE REVIEW

**Comment by domcleal** [comment]
This also needs adding to comps/.

**Comment by mmoll** [comment]
should all be fixed now

File: rubygem-ovirt_provision_plugin/rubygem-ovirt_provision_plugin.spec [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+Name:       %{?scl_prefix}rubygem-%{gem_name}
+Version:    1.0.2
+Release:    1%{?foremandist}%{?dist}
+Group:      Applications/System
+License:    GPLv3
+URL:        http://github.com/theforeman/ovirt_provision_plugin
+Source0:    http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+Requires:   foreman >= 1.11.0
+Requires:   %{?scl_prefix}rubygem(deface)
```
Comment (dLobatog) [inline rubygem-ovirt_provision_plugin/rubygem-ovirt_provision_plugin.spec]: Not sure if this should be an open dependency, I recall there were breaking changes pre 1.0 - https://github.com/spree/deface#upgrading-from-09-to-10

Comment (mmoll) [inline rubygem-ovirt_provision_plugin/rubygem-ovirt_provision_plugin.spec]: this only mirrors the open dependency in the gem

**Comment by domcleal** [comment]
Thanks @mmoll, will build for all three releases.

**Pull request merged**